client/src/components/reader/TocDrawer.js

```diff
@@ -1,5 +1,6 @@
 import React, { Component, PropTypes } from 'react';
 import { Toc } from 'components/reader';
+import ScrollLock from 'components/global/Utility/ScrollLock';
```

Comment:
Per my comment above, I'd prefer we not reference the full path, and instead import a higher level module.